data/sql/updates/pending_db_world/Horrified_Drakkari.sql

```diff
@@ -1,2 +1,2 @@
-UPDATE `creature_template` SET `flags_extra` = REPLACE(`flags_extra`, "62", "64") WHERE (`entry` = 26582);
-UPDATE `creature_template` SET `flags_extra` = REPLACE(`flags_extra`, "62", "64") WHERE (`entry` = 26583);
+UPDATE `creature_template` SET `flags_extra` = REPLACE(`flags_extra`, "64", "66") WHERE (`entry` = 26582);
+UPDATE `creature_template` SET `flags_extra` = REPLACE(`flags_extra`, "64", "66") WHERE (`entry` = 26583);
```

Comment:
change to:

```sql
UPDATE `creature_template` SET `flags_extra` = `flags_extra`|2 WHERE `entry` IN (26582, 26583);
```
by doing this you are adding that flag to the already existing `flags_extra` value. This is the conventional way to do it when you are working with a bitmask

Reply:
aka what Bench said

Reply:
see: https://www.azerothcore.org/wiki/sql-standards